packages/styled-components/src/utils/generateAlphabeticName.js

```diff
@@ -21,5 +23,7 @@ export default function generateAlphabeticName(code: number): string {
     name = getAlphabeticChar(x % charsLength) + name;
   }
 
-  return (getAlphabeticChar(x % charsLength) + name).replace(AD_REPLACER_R, '$1-$2');
+  const prefix = SC_CLASS_PREFIX ? `${SC_CLASS_PREFIX}_` : '';
```

Comment:
```suggestion
  const prefix = SC_CLASS_PREFIX ? `${SC_CLASS_PREFIX}-` : '';
```

What do you think of using a hyphen, like it's done with the "sc-" prefix?
`sc-pFZIQ`

Reply:
It's following the same pattern that the [namespace option](https://styled-components.com/docs/tooling#namespace) is using, it adds the prefix with an underscore after it

Reply:
Great, make sense for me